memreps/memreps.py

```diff
@@ -4,13 +4,13 @@
 from functools import partial
 from typing import (Any, Callable, Counter, Generator,
                     Iterable, Literal, Protocol, Union)
-
 import attr
 import exp4
 import funcy as fn
 import numpy as np
+import networkx as nx
 from exp4.algo import softmax
-
+from dfa.utils import find_subset_counterexample
```

Comment:
This should be only with DFAs right?

Reply:
yeah I don't think this is being used any longer. will fix